src/DebugBar/DataCollector/ExceptionsCollector.php

```diff
@@ -88,17 +93,11 @@ public function formatExceptionData(Exception $e)
         );
     }
 
-    /**
-     * @return string
-     */
     public function getName()
     {
         return 'exceptions';
     }
 
-    /**
-     * @return array
```

Comment:
Welp, didn't even see that. I'll get it sorted now. I blame my crazy-efficient IDE.